integration-tests/kafka-oauth/src/main/java/org/apache/camel/quarkus/kafka/oauth/it/Routes.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.kafka.oauth.it;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.apache.camel.builder.RouteBuilder;
+
+@ApplicationScoped
+public class Routes extends RouteBuilder {
+
+    @Override
+    public void configure() throws Exception {
+        // produces messages to kafka
+        from("timer:foo?period={{timer.period}}&delay={{timer.delay}}")
+                .routeId("FromTimer2Kafka")
+                .setBody().simple("Message #${exchangeProperty.CamelTimerCounter}")
+                .to("kafka:{{kafka.topic.name}}")
+                .log("Message sent correctly sent to the topic! : \"${body}\" ");
```

Comment:
Neatpick :) "Message correctly sent to the topic! : \"${body}\" " ?

Reply:
ah yes thanks